### PR TITLE
Added Create User Wine Data to the README

### LINKED ISSUE #11: README
Per the [Eval Rubric](https://backend.turing.io/module3/projects/consultancy/evaluation/presentation_documentation)
- [ ] links to other project repos & production links
- [ ] contains a list of contributors, their GitHub profiles and LinkedIn profiles
- [ ] discusses the purpose of its purpose (how it fits into the project SOA)
- [ ] discusses how to install and test the repo

Rails back-end README:
- [ ] includes database schema
- [ ] each endpoint is documented with example request & response

### PR DESCRIPTION
**Pull request by Trevorsuter**
### What Changed?

- What changes are in this PR?

**Added Create User Wine Data to the README**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,3 +107,27 @@ Things you may want to cover:
 >  }
 >}
 >```
+
+### Create User Wine Data
+`POST https://weathervine-be.herokuapp.com/api/v1/users/:user_id/wines`
```
Comment (klatour324) [inline README.md:112]: Double checking here.. do we want to have `:wine_id` at the end of the endpoint here?

**Review by klatour324 — COMMENTED** [review]
Looks good. Thanks for adding this in! Just one small clarification.

File: README.md (hunk 1 above)
Comment (Trevorsuter) [reply README.md:112]: In this case, the provided ID in the URI would be the user_wine id, which technically _isn't_ created yet (until the post is successful). We would want the id at the end of this URI for a patch request on the user wine record.

Comment (klatour324) [reply README.md:112]: Ah.. thank you for the clarification here. Yes that makes sense because we haven't created the user_wine record yet.

**Review by klatour324 — APPROVED** [review]
Thanks for the clarification! Looks good!

**Pull request merged**